volatility3/framework/plugins/windows/dumpfiles.py

```diff
@@ -44,13 +44,15 @@ def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]
                 description="Process ID to include (all other processes are excluded)",
                 optional=True,
             ),
-            requirements.IntRequirement(
+            requirements.ListRequirement(
                 name="virtaddr",
+                element_type=int,
                 description="Dump a single _FILE_OBJECT at this virtual address",
```

Comment:
This should now be these addresses, presumably?

Reply:
done